src/components/EventBus.ts

```diff
@@ -0,0 +1,35 @@
+export class EventBus {
+    _listeners: {[key: string]: Function[]};
```

Comment:
Предлагаю задать более конкретный тип для подписчика
```suggestion
    _listeners: {[key: string]: ListenerFn[]};
```
где 
```type ListenerFn = (...args?: unknown) => void```